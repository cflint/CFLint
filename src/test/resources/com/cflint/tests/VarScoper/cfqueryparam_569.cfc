<cfcomponent>
  <cffunction name="foo">
    <cfquery name="qry">
    or l1.googleID in (<cfqueryparam list="true" value="#value=arguments.placeIDs#">)
    </cfquery>
  </cffunction>
</cfcomponent>