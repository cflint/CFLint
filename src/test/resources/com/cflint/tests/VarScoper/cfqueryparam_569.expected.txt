{
  "version" : "",
  "timestamp" : 0,
  "issues" : [ {
    "severity" : "ERROR",
    "id" : "MISSING_VAR",
    "message" : "MISSING_VAR",
    "category" : "CFLINT",
    "abbrev" : "MV",
    "locations" : [ {
      "file" : "src/test/resources/com/cflint/tests/VarScoper/cfqueryparam_569.cfc",
      "fileName" : "cfqueryparam_569.cfc",
      "function" : "foo",
      "offset" : 59,
      "column" : 19,
      "line" : 3,
      "message" : "Variable qry is not declared with a var statement.",
      "variable" : "qry",
      "expression" : "<cfquery name=/"qry/">/n    or l1.googleID in (<cfqueryparam list=/"true/" value=/"#value=arguments.placeIDs#/">)/n    </cfquery>"
    } ]
  }, {
    "severity" : "ERROR",
    "id" : "MISSING_VAR",
    "message" : "MISSING_VAR",
    "category" : "CFLINT",
    "abbrev" : "MV",
    "locations" : [ {
      "file" : "src/test/resources/com/cflint/tests/VarScoper/cfqueryparam_569.cfc",
      "fileName" : "cfqueryparam_569.cfc",
      "function" : "foo",
      "offset" : 90,
      "column" : 26,
      "line" : 4,
      "message" : "Variable value is not declared with a var statement.",
      "variable" : "value",
      "expression" : "value"
    } ]
  } ],
  "counts" : {
    "totalFiles" : 0,
    "totalLines" : 0,
    "countByCode" : [ {
      "code" : "MISSING_VAR",
      "count" : 2
    } ],
    "countBySeverity" : [ {
      "severity" : "ERROR",
      "count" : 2
    } ]
  }
}